docs/core/tools/sdk-errors/netsdk1147.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "NETSDK1147: Missing workload for specified target framework"
+description: Missing workload for specified target framework
+author: tdykstra
+ms.author: tdykstra
+ms.topic: error-reference
+ms.date: 07/14/2021
+f1_keywords:
+- NETSDK1147
+---
+# NETSDK1147: Missing workload for specified target framework
```

Comment:
Great idea to add this, this will be useful 👍

Reply:
Suggestion for the product team @sfoslund: Consider an aka.ms link to this page, once live, that can be included in the error message. 

We're doing that for the Console and other template changes to help customers understand what's changing.